clinvar_this/io/gks_json.py

```diff
@@ -0,0 +1,386 @@
+"""Support for I/O of the GKS JSON format to define submissions.
+
+Currently only supports CIViC Therapeutic Assertions. This assumes you are using
+MetaKB (https://github.com/cancervariants/metakb) to generate GKS JSON files.
+"""
+
+import datetime
+import json
+import typing
+
+from ga4gh.core.models import MappableConcept
+from ga4gh.va_spec.aac_2017 import VariantTherapeuticResponseStudyStatement
+from ga4gh.va_spec.base import TherapeuticResponsePredicate, EvidenceLine, TherapyGroup
+from ga4gh.vrs.models import Syntax, Expression, Variation
+from pydantic import ConfigDict, ValidationError
+import requests
+
+from clinvar_api.models import (
+    AffectedStatus,
+    AlleleOrigin,
+    Assembly,
+    Chromosome,
+    CitationDb,
+    CollectionMethod,
+    RecordStatus,
+    SubmissionAssertionCriteria,
+    SubmissionChromosomeCoordinates,
+    SubmissionCitation,
+    SubmissionCondition,
+    SubmissionContainer,
+    SubmissionClinicalImpactSubmission,
+    SubmissionVariant,
+    SubmissionVariantSet,
+)
+from clinvar_api.models.sub_payload import (
+    SomaticClinicalImpactClassification,
+    SubmissionConditionSetSomatic,
+    SubmissionObservedInSomatic,
+    SubmissionVariantGene,
+)
+from clinvar_api.msg.sub_payload import (
+    SomaticClinicalImpactAssertionType,
+    SomaticClinicalImpactClassificationDescription,
+)
+from clinvar_this import exceptions
+from clinvar_this.io.base import TransformIO, BatchMetadata
+
+
+class BatchMetadataGksJson(BatchMetadata):
+    """Batch-wide settings for GKS JSON import.
+
+    The properties will be assigned to all variants/samples in the batch.
+    """
+
+    model_config = ConfigDict(frozen=True)
+    affected_status: typing.Optional[AffectedStatus] = None
+
+
+class GksJsonTransformer(TransformIO):
+    """Class for transforming GKS JSON input data from various formats into submission format"""
+
+    batch_metadata_defaults = BatchMetadataGksJson(
+        collection_method=CollectionMethod.CURATION,
+        allele_origin=AlleleOrigin.SOMATIC,
+        affected_status=AffectedStatus.YES,
```

Comment:
Excel sheet has yes, but https://www.ncbi.nlm.nih.gov/clinvar/variation/13961/?oq=SCV004565362.1 has unknown?

Reply:
Changed to unknown, lmk if I should revert